app/scripts/modules/azure/serverGroup/serverGroup.transformer.js

```diff
@@ -3,57 +3,53 @@
 let angular = require('angular');
 
 module.exports = angular.module('spinnaker.azure.serverGroup.transformer', [
-    require('../../core/utils/lodash.js'),
-    require('../vpc/vpc.read.service.js'),
   ])
-  .factory('azureServerGroupTransformer', function (_, azureVpcReader) {
+  .factory('azureServerGroupTransformer', function () {
 
-    function normalizeServerGroup(serverGroup) {
-      serverGroup.instances.forEach((instance) => { instance.vpcId = serverGroup.vpcId; });
-      return azureVpcReader.listVpcs().then(addVpcNameToServerGroup(serverGroup));
-    }
-
-    function addVpcNameToServerGroup(serverGroup) {
-      return function(vpcs) {
-        var matches = vpcs.filter(function(test) {
-          return test.id === serverGroup.vpcId;
-        });
-        serverGroup.vpcName = matches.length ? matches[0].name : '';
-        return serverGroup;
+    function convertServerGroupCommandToDeployConfiguration(base) {
+      var command = {
+        name: base.application,
+        cloudProvider: base.selectedProvider,
+        application: base.application,
+        stack: base.stack,
+        detail: base.details,
+        credentials: base.credentials,
+        region: base.region,
+        user: '[anonymous]',
+        upgradePolicy: 'Manual',
+        type: 'createServerGroup',
+
+        image: {
+          publisher: 'Canonical',
+          offer: 'UbuntuServer',
+          sku: '15.10',
+          version: 'latest',
+        },
+
+        sku: {
+          name: 'Standard_A1',
+          tier: 'Standard',
+          capacity: 2,
+        },
+
+        osConfig: {
+          adminUserName: 'spinnakeruser',
+          adminPassword: '!Qnti**234',
+        },
       };
-    }
 
-    function convertServerGroupCommandToDeployConfiguration(base) {
-      // use _.defaults to avoid copying the backingData, which is huge and expensive to copy over
-      var command = _.defaults({backingData: [], viewState: []}, base);
-      if (base.viewState.mode !== 'clone') {
-        delete command.source;
-      }
-      if (base.viewState.useAllImageSelection) {
-        command.amiName = base.viewState.allImageSelection;
+      if (typeof base.stack != 'undefined') {
+        command.name = command.name + '-' + base.stack;
       }
-      command.availabilityZones = {};
-      command.availabilityZones[command.region] = base.availabilityZones;
-      command.account = command.credentials;
-      if (!command.ramdiskId) {
-        delete command.ramdiskId; // TODO: clean up in kato? - should ignore if empty string
-      }
-      delete command.region;
-      delete command.viewState;
-      delete command.backingData;
-      delete command.selectedProvider;
-      delete command.instanceProfile;
-      delete command.vpcId;
-
-      if (!command.subnetType) {
-        delete command.subnetType;
+      if(typeof base.details != 'undefined') {
```

Comment:
I think this will cause a couple of lint warnings (space between `if` and `(`, `!=` instead of `!==`)